docs/guides/quick-start.md

```diff
@@ -0,0 +1,147 @@
+```
+
+Soon after that your Boss comes and asks you to implement API for that model so that everyone can access our great fruit database from all over the world.
+
+Fruit model has name and color attributes and we want to publish them both. GraphQL output type for our model is generated by using `strawberry_django.type` decorator. Both fields are char fields which means that we want to use python `str`type for them.
+
+```python
+# types.py
+import strawberry_django
+from . import models
+
+@strawberry_django.type(models.Fruit)
+class Fruit:
+    name: str
+    color: str
+```
+
+Last step is to generate Query type and Schema. Our big brother `strawberry` helps us here.
+
+```python
+# schema.py
+import strawberry
+from typing import List
+from .types import Fruit
+
+@strawberry.type
+class Query:
+    fruits: List[Fruit]
+
+schema = strawberry.Schema(query=Query)
+```
+
+Finally we add `AsyncGraphQLView` view into our urls so that we can start making first queries.
+```python
+# urls.py
+from django.urls import include, path
+from strawberry.django.views import AsyncGraphQLView
+from .schema import schema
+
+urlpatterns = [
+    path('graphql', AsyncGraphQLView.as_view(schema=schema)),
+]
+```
+
+After that, once the development server is running, you can read your fruits from the database.
+
+```graphql
+query {
+  fruits { name color }
+}
+# -> fruits: [{ name: "strawberry", color: "red" }]
+```
+
+## Model Relations
+
+Let's create another model called `Color` and add foreign key relation between `Fruit` and `Color` models.
+
+```python
+# models.py
+from django.db import models
+
+class Fruit(models.Model):
+    name = models.CharField(max_length=20)
+    color = models.ForeignKey('Color', related_name='fruits', on_delete=models.CASCADE)
+
+class Color(models.Model):
+    name = models.CharField(max_length=20)
+```
+
+We also need to add another type and modify existing one. Field type `auto` is used for type auto resolution. `strawberry_django` goes through all fields and resolves field types. It also generates resolvers for relation fields on behalf you.
+
+```python
+# types.py
+import strawberry_django
+from strawberry_django import auto
+from typing import List
+from . import models
+
+@strawberry_django.type(models.Fruit)
+class Fruit:
+    id: auto
+    name: auto
+    color: 'Color'
+
+@strawberry_django.type(models.Color)
+class Color:
+    id: auto
+    name: auto
+    fruits: List[Fruit]
+```
+
+This generates schema like this
+```graphql
+type Color {
+  id: ID!
+  name: String!
+  fruits: [Fruit!]
+}
+
+type Fruit {
+  id: ID!
+  name: String!
+  color: Color!
+}
+
+type Query {
+  fruits: [Fruit!]!
+}
+```
+
+Now you can start making queries and request all fruits and their colors from the database.
+
+```graphql
+query {
+  fruits {
+    name
+    color {
+      name
+    }
+  }
+}
+# fruits: [
+#   { name: "strawberry", color: { name: "red" } },
+#   { name: "raspberry", color: { name: "red" } }
+# ]
+```
```

Comment:
Would be useful to add mutation here? Or you are leaving that out because API is not definitive yet?

Reply:
> Nice work!

Thanks :) and thanks for your feedback. Really appreciate it.


>I do think it would be good to use [black](https://github.com/psf/black)

That's probably good idea. I would leave that out from this massive pull request.

Reply:
There is separate page for mutations but it's good idea to say something about mutations in quick start as well. Documentation is one big thing we really need to focus on soon.